cmd/fileshare_process/main.go

```diff
@@ -0,0 +1,108 @@
+// NordVPN fileshare daemon.
+package main
+
+import (
+	"errors"
+	"log"
+	_ "net/http/pprof" // #nosec G108 -- http server is not run in production builds
+	"os"
+	"os/user"
+	"path/filepath"
+	"strconv"
+
+	"github.com/NordSecurity/nordvpn-linux/fileshare_process"
+	"github.com/NordSecurity/nordvpn-linux/fileshare_startup"
+	"github.com/NordSecurity/nordvpn-linux/internal"
+)
+
+// Values set when building the application
+var Environment = ""
+
+func openLogFile(path string) (*os.File, error) {
+	// #nosec path is constant
+	logFile, err := os.OpenFile(path, os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0600)
+	if err != nil {
+		return nil, err
+	}
+	return logFile, nil
+}
+
+func main() {
+	if logFile, err := openLogFile(filepath.Join(fileshare_process.FileshareLogPath, "/nordfileshared.log")); err == nil {
```

Comment:
is there any file rotation in place or something similar to restrict log file at a certain size?

Reply:
We don't do rotation for local user logs afaik, probably should implement that.